lib/llm-events/aws-bedrock/utils.js

```diff
@@ -67,7 +69,63 @@ function stringifyConverseChunkedMessage(chunks) {
   return stringifiedChunks.join('\n\n')
 }
 
+/**
+ * Set the prompt, completion and total token counts on the Llm Event if input and output metadata
+ * exists in the response
+ * @param {object} response - The response object.
+ * @param {object} context - The Llm Event
+ */
+function setUsageTokens(response, context) {
+  // input and output token counts must available in order to add all usage attributes to response
+  // if total tokens is not available, we can manually add it up (from input and output token count)
+  if (tokenUsageAttributesExist(response)) {
+    const promptTokens =
+      Number(response?.usage?.input_tokens ||
+       response?.usage?.inputTokens ||
+        response?.parsedBody?.results?.[0]?.inputTextTokenCount ||
+         response?.parsedBody?.usage.input_tokens)
```

Comment:
Given the variety in how these are defined, we're on the hook for *some* kind of complex conditional somewhere. I wonder if these could be broken out into helper functions, though. And we may want to allow for `response.usage` to be undefined in some of these.